src/routers/DocsRouter.ts

```diff
@@ -29,6 +32,12 @@ const openAPISpec = routingControllersToSpec(rcMetadataStorage, rcOptions, {
   },
 });
 
+openAPISpec.components.securitySchemes = {};
+openAPISpec.components.securitySchemes[SecuritySchemeName] = {
+  type: 'http',
+  scheme: 'bearer',
```

Comment:
What is 'bearer'?

Reply:
https://swagger.io/docs/specification/authentication/bearer-authentication/
will add comment